http/vertx/pom.xml

```diff
@@ -31,7 +31,7 @@
     <packaging>jar</packaging>
 
     <properties>
-        <vertx.version>3.9.2</vertx.version>
+        <vertx.version>4.0.0.Beta1</vertx.version>
```

Comment:
Any specfic reason to not use a `miltestone*`? (https://mvnrepository.com/artifact/io.vertx/vertx-core)

Reply:
Vert.x 4 already went through a bunch of milestones round, Beta1 is the next phase of the release